Rakefile

```diff
@@ -581,7 +581,7 @@ If you installed node without using 'brew install node', follow these instructio
       package = {header: package_name}.merge(package)
       system "clear"
       until result = verify_package(package_name, package)
-        notify "\n#{'-' * 25}\nWARNING: This package is NOT installed correctly.  Follow the instructions below.  \nIf issues persist, raise your hand and someone will assist you."
+        notify "\n#{'-' * 25}\nIT'S YOUR TURN! Either you haven't installed this application yet, or it needs some tweaking. No worries! Just follow the steps below. :)\nIf issues persist, raise your hand and an instructor will assist you."
```

Comment:
"Just follow the steps below, pasting the commands into the OTHER terminal window."